ui/config-overrides.js

```diff
@@ -1,3 +1,13 @@
-const { useBabelRc, override, useEslintRc } = require('customize-cra');
+const {
```

Comment:
Hum, wrong commit?

Looks like this change belongs to "Add compression-webpack-plugin to create gziped file durint the build" instead of "Add gzip to nginx conf".